### PR TITLE
Add/Update AWS Audit Manager > HIPAA Final Omnibus Security Rule 2013 controls. Closes #621

### LINKED ISSUE #621: Add/Update AWS Audit Manager > HIPAA Final Omnibus Security Rule 2013 controls (2023-05-09)
**Is your feature request related to a problem? Please describe.**

**New Additions**

Benchmark | Control(s)
-- | --
164_308_a_3_i | ssm_document_not_public
164_308_a_3_ii_b | ssm_document_not_public
164_308_a_4_i | ssm_document_not_public
164_308_a_4_ii_b | ssm_document_not_public
164_308_a_4_ii_c | ssm_document_not_public
164_312_a_1 | ssm_document_not_public
164_312_e_1 | ssm_document_not_public
164_314_b_1 | service_vpc_endpoint_enabled
164_314_b_2 | service_vpc_endpoint_enabled
164_314_b_2_i | service_vpc_endpoint_enabled
164_314_b_2_ii | service_vpc_endpoint_enabled
164_314_b_2_iii | service_vpc_endpoint_enabled
164_314_b_2_iv | service_vpc_endpoint_enabled

**Removed Controls**

Benchmark | Control (s)
-- | --
164_308_a_4_ii_b | iam_root_user_no_access_keys
164_308_a_4_ii_c | s3_public_access_block_account
164_312_a_1 | opensearch_domain_in_vpc
164_312_b | apigateway_rest_api_stage_xray_tracing_enabled
164_312_c_1 | opensearch_domain_node_to_node_encryption_enabled
164_312_e_1 | opensearch_domain_node_to_node_encryption_enabled
164_312_e_1 | s3_public_access_block_account
164_314_b_1 | vpc_endpoint_service_acceptance_required_enabled
164_314_b_2 | vpc_endpoint_service_acceptance_required_enabled
164_314_b_2_i | vpc_endpoint_service_acceptance_required_enabled
164_314_b_2_ii | vpc_endpoint_service_acceptance_required_enabled
164_314_b_2_iii | vpc_endpoint_service_acceptance_required_enabled
164_314_b_2_iv | vpc_endpoint_service_acceptance_required_enabled

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Priyanka-Chatterjee-2000**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_1.sp [hunk 1]
```diff
@@ -27,7 +27,7 @@ benchmark "hipaa_final_omnibus_security_rule_2013_164_314_b_1" {
     control.s3_public_access_block_account,
     control.sagemaker_endpoint_configuration_encryption_at_rest_enabled,
     control.sagemaker_notebook_instance_encryption_at_rest_enabled,
-    control.vpc_endpoint_service_acceptance_required_enabled
+    control.vpc_configured_to_use_vpc_endpoints
```
Comment (khushboo9024) [inline hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_1.sp:30]: Please remove tag `hipaa_final_omnibus_security_rule_2013 = "true"` from control and please validate for other removed controls as well.

File: hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_2.sp [hunk 2]
```diff
@@ -27,7 +27,7 @@ benchmark "hipaa_final_omnibus_security_rule_2013_164_314_b_2" {
     control.s3_public_access_block_account,
     control.sagemaker_endpoint_configuration_encryption_at_rest_enabled,
     control.sagemaker_notebook_instance_encryption_at_rest_enabled,
-    control.vpc_endpoint_service_acceptance_required_enabled
+    control.vpc_configured_to_use_vpc_endpoints
```
Comment (khushboo9024) [inline hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_2.sp:30]: @Priyanka-Chatterjee-2000 Please add tag `hipaa_final_omnibus_security_rule_2013 = "true"` to the control and check for other added controls as well.

File: conformance_pack/cloudtrail.sp [hunk 3]
```diff
@@ -176,8 +176,8 @@ control "cloudtrail_s3_logging_enabled" {
   query       = query.cloudtrail_s3_logging_enabled
 
   tags = merge(local.conformance_pack_cloudtrail_common_tags, {
-    gdpr     = "true"
-    nist_csf = "true"
+    gdpr                                   = "true"
+    nist_csf                               = "true"
```
Comment (madhushreeray30) [inline conformance_pack/cloudtrail.sp:180]:
```suggestion
    gdpr     = "true"
    nist_csf = "true"
```

File: hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_1.sp
Comment (madhushreeray30) [inline hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_1.sp:1]: please add `hipaa_final_omnibus_security_rule_2013 = "true"` tag for control `vpc_configured_to_use_vpc_endpoints`

**Review by madhushreeray30 — CHANGES_REQUESTED** [review]
@Priyanka-Chatterjee-2000 please look at the review comments, thanks!

**Comment by madhushreeray30** [comment]
@Priyanka-Chatterjee-2000 please resolve the merge conflits.

**Review by madhushreeray30 — APPROVED** [review]
LGTM 👍

**Review by misraved — APPROVED** [review]

**Pull request merged**